src/flag_gems/ops/gather.py

```diff
@@ -258,3 +260,8 @@ def gather(inp, dim, index, out=None, sparse_grad=False):
 
     _gather_func(inp_strided, out, index, dim, stride_dim, M, N)
     return out
+
+
+def gather_backward(grad, self, dim, index, sparse_grad):
```

Comment:
Please add log here and check whether the relevant code is executed when running the unit test, because the coverage CI does not pass

Reply:
Thanks, I added the log.